### PR TITLE
tautulli: 2.2.4 -> 2.6.1

### PR DESCRIPTION
**Pull request by bachp**
###### Motivation for this change

Remove python2 dependency and update to latest version.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 100518` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>tautulli</li>
  </ul>
</details>

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -6493,7 +6493,7 @@ in
 
   tab = callPackage ../tools/text/tab { };
 
-  tautulli = callPackage ../servers/tautulli { python = python2; };
+  tautulli = callPackage ../servers/tautulli { python = python3; };
```
Comment (SuperSandro2000) [inline pkgs/top-level/all-packages.nix:6496]: Can we just pass in python3?

Comment (bachp) [reply pkgs/top-level/all-packages.nix:6496]: I don't see why not. I will change the PR accordingly

**Comment by lukegb** [comment]
Might be worth closing this in favour of #104496, which does some other cleanups as well.

**Comment by bachp** [comment]
Closing because https://github.com/NixOS/nixpkgs/pull/104496 was merged.

**Pull request closed without merging**